_episodes/05-memory.md

```diff
@@ -7,8 +7,11 @@ questions:
 - "What is cognitive load and how does it affect learning?"
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Identify when cognitive load is likely to limit learning."
+- "Remember the quantitative limit of human memory." 
+- "Identify the number of concepts required in a lesson by drawing a concept map." 
 - "Use a concept map to plan a formative assessment strategy."
+- "Describe the three components of cognitive load theory and which should be emphasized or diminished." 
```

Comment:
While we do cover the three components specifically, we don't have an activity that asks learners to describe them. I find that most learner conversations focus on extraneous load and don't lean too heavily on the sub-terms (i.e. use "add to cognitive load" vs "create extraneous load").

Reply:
I changed "Describe" to "Name"